src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/BotSegments/BotSegmentsPlaybackController.cs

```diff
@@ -224,7 +224,8 @@ public void Stop()
             }
 
             _nextBotSegments.Clear();
-            _isPlaying = PlayState.NotLoaded;
+            _playState = PlayState.NotLoaded;
+            BotSequence.ActiveBotSequence = null;
```

Comment:
I think this may have been the change that fixed my issue. Thanks!

Reply:
Context for others seeing this comment, the issue I was seeing is that after I ran or stopped a sequence, it wouldn't allow me to start a new sequence because it thought I still had an active sequence.